hbase-common/src/main/java/org/apache/hadoop/hbase/util/ReflectionUtils.java

```diff
@@ -228,4 +229,38 @@ public static Object invokeMethod(Object obj, String methodName, Object... param
     return parameterTypes;
   }
 
+  public static Field getModifiersField() throws IllegalAccessException, NoSuchFieldException {
+    // this is copied from https://github.com/powermock/powermock/pull/1010/files to work around
+    // JDK 12+
```

Comment:
This is for jdk17 or jdk11? The title of the PR says this is jdk11 but the comment here is for jdk12+...

Reply:
looks like it issues a warning in JDK11 and throws exception in JDK12 and above. I can update the subject